migration/migration/006-matcher-operators.sql

```diff
@@ -128,7 +131,33 @@ AS $func$
     FROM _prom_catalog.label l
     WHERE l.key = key_to_match and l.value ~ pat
 $func$
-LANGUAGE SQL STABLE PARALLEL SAFE;
+LANGUAGE SQL STABLE PARALLEL SAFE
+SUPPORT _prom_ext.rewrite_fn_call_to_subquery;
+
+-- TODO (james): should these operators be in the _prom_catalog schema, or a "public" schema?
+CREATE OPERATOR _prom_catalog.== (
+    LEFTARG = prom_api.label_key,
+    RIGHTARG = prom_api.pattern,
+    FUNCTION = _prom_catalog.label_find_key_equal
+);
+
+CREATE OPERATOR _prom_catalog.!== (
+    LEFTARG = prom_api.label_key,
+    RIGHTARG = prom_api.pattern,
+    FUNCTION = _prom_catalog.label_find_key_not_equal
+);
+
+CREATE OPERATOR _prom_catalog.==~ (
+    LEFTARG = prom_api.label_key,
+    RIGHTARG = prom_api.pattern,
+    FUNCTION = _prom_catalog.label_find_key_regex
+);
+
+CREATE OPERATOR _prom_catalog.!=~ (
+    LEFTARG = prom_api.label_key,
+    RIGHTARG = prom_api.pattern,
+    FUNCTION = _prom_catalog.label_find_key_not_regex
+);
```

Comment:
we don't need any of these operator definitions anymore since this is now done differently here https://github.com/timescale/promscale_extension/blob/develop/migration/idempotent/002-tag-operators.sql#L34

Delete all of this please.

Reply:
done